eigenda/commitment.go

```
@@ -35,6 +36,11 @@ func (c Commitment) Encode() []byte {
 	return append([]byte{byte(EigenDA), byte(EigenV0)}, c...)
 }
 
+func StringToCommit(key string) (Commitment, error) {
+	comm, _ := hexutil.Decode(key)
```

any reason why we're implicitly ignoring the err?